### PR TITLE
Fixes on CourseOverviewTab

### PR DESCRIPTION
**Pull request by y2kwak**
## Implementation Description
Display spinner on loading
Display "There are no courses" text instead of dummyData when there are no courses.

## Screenshots
Screen Shot 2022-03-27 at 15 57 55
Screen Shot 2022-03-27 at 15 58 55 (2)

## What should reviewers focus on?
- 

## Testing Procedure
- What test(s) should we run to make sure your code works?
   - What ways could the input be wrong?
   - Is it secure? Could nefarious users access/break it?

## Things to Note
- Additional dependencies/libraries you've added?
- Things you'd want to know when coming back to the code after a few months

## Checklist
- [x] Ensure code follows style guide by running the linters
- [ ] I have updated the documentation or deemed it unnecessary
- [x] I have linked the relevant issue in this PR
- [x] I have requested a review from the PL, as well as other dev(s) (and designers if necessary)

### CODE REVIEW

File: frontend/src/components/pages/AdminDashboard.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import CoursesOverviewTab from "../admin/CoursesOverviewTab";
 
 const AdminDashboard = (): React.ReactElement => {
   return (
-    <Flex>
+    <Flex h="100vh">
```
Comment (e-wai) [inline frontend/src/components/pages/AdminDashboard.tsx:8]: Ah very smart 👀

**Review by e-wai — APPROVED** [review]
What are your thoughts on moving the spinner into the center of the screen? Rather than at the top of container
- Having a "no elements here" card would also be nice under the "Modules" title on each `CoursePreview`

**Review by draymond63 — APPROVED** [review]
Looks good to me, although I think it could be worth extracting the course preview logic into a separate function like Emma said.

File: frontend/src/components/admin/CoursesOverviewTab.tsx [hunk 2]
```diff
@@ -39,16 +37,21 @@ const CoursesOverviewTab = (): React.ReactElement => {
...
+            <CoursePreview
+              key={course.id}
+              courseId={course.id}
+              title={course.title}
+              description={course.description}
+              isPrivate={course.private}
+              modules={course.modules}
+            />
+          ))
+        )}
```
[the remark this answers is not in the document]
Comment (y2kwak) [reply frontend/src/components/admin/CoursesOverviewTab.tsx:54]: yes! i like that idea

**Pull request merged**